gramps/plugins/textreport/descendreport.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python
```

Comment:
There is no need to add this.

Reply:
1. Some .py files in Gramps have:
#!/usr/bin/env python
Some files have:
# -*- coding: utf-8 -*-
Some files not have nothing in it's header.
Usually all source in project use one header for unification (l' uniformité).

2. Gramps can run on Linux, Windows, Mac.
As you know - in Russia OS Windows can use **CP-1251** for GUI and **cp866** for Command line. 

3. And on https://docs.python.org/3/howto/unicode.html and https://stackoverflow.com/questions/14083111/should-i-use-encoding-declaration-in-python-3 I see - 'this header need'.

Reply:
@PushKK The coding comment is no longer needed for Python3 but as the SO article points out it's still useful for some editors. The format originated in emacs but there are other editors that understand it. The python comment is used to allow those files to be executed directly from a shell on Unix systems. Neither is anything that translators need to worry about.

No, Windows can not use cp1251--or any other antique code page--for the native GUI. All internal strings in Windows are encoded in UTF16. Windows still uses code pages for the CMD shell and Powershell, though they're deprecated in favor of UTF8 in the latest version of Powershell. Gramps however doesn't use the native GUI, it uses [Gtk](https://www.gtk.org) and that requires that strings be encoded in UTF8, so all translation (i.e. po) files need to be encoded in UTF8. You don't need to worry about that either as long as you use weblate because it will take care of the encoding for you.

Reply:
Thank you. My last work with Windows had many problems with codepages.
MS Windows 7 Pro can use Unicode, software from Germany use CP-1252, our customer's software use CP-1251.
Every update software can was causing problems.